cmd/metal-api/internal/service/network-service.go

```diff
@@ -409,6 +417,23 @@ func (r *networkResource) createNetwork(request *restful.Request, response *rest
 	r.send(request, response, http.StatusCreated, v1.NewNetworkResponse(nw, usage))
 }
 
+func validateAdditionalAnnouncableCIDRs(additionalCidrs []string, privateSuper bool) ([]string, error) {
+	var result []string
+	if len(additionalCidrs) > 0 {
+		if !privateSuper {
+			return nil, errors.New("additionalannouncablecidrs can only be set in a private super network")
+		}
+		for _, cidr := range additionalCidrs {
+			_, err := netip.ParsePrefix(cidr)
+			if err != nil {
+				return nil, fmt.Errorf("given cidr:%q in additionalannouncablecidrs is malformed:%w", cidr, err)
+			}
+			result = append(result, cidr)
+		}
+	}
```

Comment:
Indent level can be reduced:

```suggestion
	if len(additionalCidrs) == 0 {
	  return nil
	}
	
        if !privateSuper {
		return nil, errors.New("additionalannouncablecidrs can only be set in a private super network")
	}
	
	...
```

Reply:
yes, done